src/test/java/CodeTest/classtest2.java

```diff
@@ -0,0 +1,170 @@
+package CodeTest;
+
+import Helper.Misc;
+import Web.MyDriver;
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class classtest2 {
+
+    /**
+     * Ques-2:
+     * Verify feelsLike-temp value is in between the low-temp value and high-temp value
+     * Web: https://www.darksky.net/
+     * points: 20
+     */
+
+    @Test
+    public void seleniumQuestion() {
+        MyDriver.launchUrlOnNewWindow("https://darksky.net/");
+
+        String feelTemp = MyDriver.getDriver().findElement(By.xpath("//*[@class='feels-like-text']")).getText();
+        String lowTemp = MyDriver.getDriver().findElement(By.xpath("//*[@class='low-temp-text']")).getText();
+        String highTemp = MyDriver.getDriver().findElement(By.xpath("//*[@class='high-temp-text']")).getText();
+
+
+        int newFeelsTemp = Integer.valueOf(feelTemp.split("˚")[0]);
+        int newLowTemp = Integer.valueOf(lowTemp.split("˚")[0]);
+        int newHighTemp = Integer.valueOf(highTemp.split("˚")[0]);
+
+        Assert.assertTrue(newFeelsTemp > newLowTemp && newFeelsTemp < newHighTemp);
```

Comment:
Also add a message , in case assertion is not correct

Reply:
Assert.assertTrue(newFeelsTemp > newLowTemp && newFeelsTemp < newHighTemp, "not correct values are shown");